io/disk/dlpar_vscsi.py

```diff
@@ -57,15 +38,16 @@ def setUp(self):
         self.vios_ip = self.params.get('vios_ip', '*', default=None)
         self.vios_user = self.params.get('vios_username', '*', default=None)
         self.vios_pwd = self.params.get('vios_pwd', '*', default=None)
-        self.login(self.vios_ip, self.vios_user, self.vios_pwd)
+        self.session = Session(self.vios_ip, user=self.vios_user,
+                               password=self.vios_pwd)
```

Comment:
So in all the test cases, does it require to close this session ?  should this be taken care in teardown ?

Reply:
not needed

Reply:
not needed this.